src/UnitTests/Core/CustomCommandTests/GetCommandsToRun_Should.cs

```diff
@@ -10,12 +12,16 @@ namespace ChatterBot.Tests.Core.CustomCommandTests
 {
     public class GetCommandsToRun_Should
     {
-        private readonly CommandsSet _commandsSet;
+        private readonly ICommandsSet _commandsSet;
         private readonly CustomCommand _customCommand;
 
         public GetCommandsToRun_Should()
         {
-            _commandsSet = new CommandsSet();
+            var fakeAppSettings = new ApplicationSettings() { Entropy = "SomeFakedEntropyString", LightDbConnection = "Filename=database.db;Password=1234" };
```

Comment:
If we want to avoid having the db file persist, we can make it temporary or have the connection string keep the db in memory only.

From the LiteDb docs regarding the `Filename` part of the `ConnectionString`.
>Full or relative path to the datafile. Supports :memory: for memory database or :temp: for in disk temporary database (file will deleted when database is closed)